app/views/admin/tags/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with model: [:admin, tag], data: { remote: false } do |f| %>
+<%= form_with model: [:admin, tag], local: true do |f| %>
```

Comment:
these are functionally equivalent but `local: true` is more "Railsy", see https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-form_with

Reply:
Isn't `local: true` the default? 

>  By default form submits via typical HTTP requests. Enable remote and unobtrusive XHRs submits with `local: false`